exporter/coralogixexporter/exporter.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coralogixexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/coralogixexporter"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/model/pdata"
+	"go.uber.org/zap"
+)
+
+// CoralogixExporter by Coralogix
+type CoralogixExporter struct {
+	cfg    Config
+	logger *zap.Logger
```

Comment:
You might want to store a telemetrysettings here instead, like this:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/eacfde3fcbd46ba60a6db0e9a41977390c4883bd/exporter/skywalkingexporter/skywalking.go#L46

It contains a logger, a tracer, and a meter.

Reply:
Thanks, but in this time we not use component.TelemetrySettings